src/drc/api/apps.py

```diff
@@ -1,9 +1,13 @@
 from django.apps import AppConfig
 
+from vng_api_common.api import register_extensions
+
 
 class DRCApiConfig(AppConfig):
     name = "drc.api"
 
     def ready(self):
         # ensure that the metaclass for every viewset has run
         from . import views  # noqa
+
+        register_extensions()
```

Comment:
```suggestion
    def ready(self):
        register_extensions()
        
        # ensure that the metaclass for every viewset has run
        from . import views  # noqa
```
Om er zeker van te zijn dat ze zo vroeg mogelijk ingeladen worden